subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/projectmodule/DefaultProjectDependencyPublicationResolver.java

```diff
@@ -79,17 +79,19 @@ public <T> T resolve(Class<T> coordsType, ProjectInternal project) {
         // Select all entry points. An entry point is a publication that does not contain a component whose parent is also published
```

Comment:
🧐 This block really ought to be extracted to its own method.

Reply:
Extracted